databases/mydb/migrations/03_add_bookmarks.up.sql

```diff
@@ -0,0 +1,49 @@
+------------
+-- Tables --
+------------
+
+DROP TABLE IF EXISTS folders;
+CREATE TABLE folders
+(
+    id         BIGSERIAL PRIMARY KEY,
+    name       VARCHAR(256),
+    user_id    BIGINT REFERENCES users                NOT NULL,
+    created_at TIMESTAMP WITH TIME ZONE DEFAULT now() NOT NULL
+);
+
+DROP TABLE IF EXISTS bookmarks;
+CREATE TABLE bookmarks
+(
+    id          BIGSERIAL PRIMARY KEY,
+    name        VARCHAR(256),
+    user_id     BIGINT REFERENCES users                NOT NULL,
+    source_uid  CHAR(8)                                NOT NULL,
+    source_type VARCHAR(32)                            NOT NULL,
```

Comment:
what is source_type ?

Reply:
content type of source (SOURCE, LIKUTIM ...)
loo like need to rename from source

Reply:
maybe stick with `content_type` ?

Reply:
I will go to this
https://github.com/Bnei-Baruch/archive-my/blob/55078023f1c59ea5d2d83b7e05f41f8daf20cc11/databases/mydb/migrations/01_init_mydb.up.sql#L49

Reply:
so make it `subject_uid` as well. for consistency